public/css/list/list-item.less

```diff
@@ -121,6 +121,10 @@
     .status-icons {
       display: flex;
       align-items: center;
+
+      .flapping-icon:not(:first-child) {
+        margin-left: .3em;
```

Comment:
Please remove this. It looks bad and unnecessary:
![Screenshot from 2021-10-01 14-23-46](https://user-images.githubusercontent.com/16668527/135619208-1c9b253f-6566-42a2-b578-c0a0e6de5154.png)